doc/classes/Array.xml

```diff
@@ -305,7 +305,7 @@
 		</method>
 		<method name="sort">
 			<description>
-				Sort the array using natural order.
+				Sorts the array using natural order.
```

Comment:
This description is misleading. Sorting in [natural order](https://en.wikipedia.org/wiki/Natural_sort_order) means sorting strings in a human-friendly way but that's not what this method does.
```gdscript
var a = ['22 days', '2 days', '17 days']
a.sort()
print(a) # => [17 days, 2 days, 22 days] but with natural order would be [2 days, 17 days, 22 days]
```

Reply:
That's a good catch! I don't think I've ever needed to sort a string array like that. What do we think is a better description here: "Sorts the array. Note: strings are sorted in alphabetical, not natural order."

Reply:
I believe the technical term is lexicographical order.

Reply:
"alphabetical" is a more common synonym, so I think we'll stick with that.